woocommerce/woocommerce-admin
# Add progress bars to Historical Data Import screen

Fixes #1851.

This PR hooks up `import/status` endpoint to _Historical Data Import_ screen.

### Accessibility




### Screenshots
![image](https://user-images.githubusercontent.com/3616980/58426966-78b7a480-809e-11e9-912f-155c4bea8f79.png)

### Detailed test instructions:
- Go to _Settings_ and _Import Historical Data_.
- Start an import.
- Verify progress bars update and reflect the progress status.
- Try pausing an import, deleting previous data, changing the period, reloading the page while an import is running, etc. Verify no errors are shown and page behavior matches the designs here:
https://www.figma.com/proto/G1xF07y9TuuhfirzYtqmIvTO/Analytics-Settings
